eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json

```diff
@@ -0,0 +1,639 @@
+{
+  "process_graph": {
+    "1": {
+      "process_id": "merge_cubes",
+      "arguments": {
+        "cube2": {
+          "from_node": "31"
+        },
+        "cube1": {
+          "from_node": "10"
+        }
+      }
+    },
+    "2": {
+      "process_id": "merge_cubes",
+      "arguments": {
+        "cube2": {
+          "from_node": "30"
+        },
+        "cube1": {
+          "from_node": "11"
+        }
+      }
+    },
+    "3": {
+      "process_id": "merge_cubes",
+      "arguments": {
+        "cube2": {
+          "from_node": "36"
+        },
+        "cube1": {
+          "from_node": "34"
+        }
+      }
+    },
+    "4": {
+      "process_id": "merge_cubes",
+      "arguments": {
+        "cube2": {
+          "from_node": "25"
+        },
+        "cube1": {
+          "from_node": "35"
+        },
+        "overlap_resolver": {
```

Comment:
Here's still a multiply overlap_resolver. Is that intentional?

Reply:
No I just forgot it, let me remove it.